junit-launcher/src/main/java/org/junit/gen5/launcher/main/Root.java

```diff
@@ -44,15 +50,13 @@ TestDescriptor getTestDescriptorFor(TestEngine testEngine) {
 	}
 
 	void applyPostDiscoveryFilters(TestDiscoveryRequest discoveryRequest) {
-		TestDescriptor.Visitor filteringVisitor = (descriptor, remove) -> {
-			if (!descriptor.isTest())
-				return;
-
-			if (isExcluded(discoveryRequest, descriptor)) {
+		Filter<TestDescriptor> postDiscoveryFilter = composeFilters(discoveryRequest.getPostDiscoveryFilters());
```

Comment:
FYI: it's not possible to get the formatting wrong in general. Any violations will result in a broken build... which we will not accept.